client/platform/web-girder/views/Upload.vue

```diff
@@ -289,6 +293,14 @@ export default defineComponent({
         disabled: uploading.value,
       };
     });
+    const errorHandler = async ({ err, name }: {err: AxiosError; name: string}) => {
```

Comment:
in try/catch, errors are always "any", so we can't know that this is an axios error without a type assertion.  This is why `getResponseError` has a fallback to returning the param as-is.  Could you change `err` to be `any` instead?